cmd/kube-scheduler/app/options/options.go

```diff
@@ -156,7 +143,7 @@ func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 	fs.StringVar(&o.Master, "master", o.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
 
 	o.SecureServing.AddFlags(nfs.FlagSet("secure serving"))
-	o.CombinedInsecureServing.AddFlags(nfs.FlagSet("insecure serving"))
+	addDummyInsecureFlags(nfs.FlagSet("insecure serving"))
 	o.Authentication.AddFlags(nfs.FlagSet("authentication"))
 	o.Authorization.AddFlags(nfs.FlagSet("authorization"))
 	o.Deprecated.AddFlags(nfs.FlagSet("deprecated"), &o.ComponentConfig)
```

Comment:
maybe have them as part of `o.Deprecated`?

Reply:
Can you more elaborate on what you mean? What is "them" in the context? flags set through `addDummyInsecureFlags`?

Reply:
the insecure flags can be part of `o.Deprecated`, so that we keep them in a single place.

Reply:
Done